### DISCUSSION TITLE
Experimental-Ender3V2-427-BLTUBL-IS-MPC-20221222

### DISCUSSION THREAD

**Discussion by JEBurroughs** [mriscoc/Ender3V2S1#569] [Show and tell]
I have 2 Aquila frames that I'm running Ender 4.2.7 mainboards in. I used the new Experimental-Ender3V2-427-BLTUBL-IS-MPC-20221222 version with input shaping and MPC temp control.  My first print was a 7 plus hour Vase mode Rose and I have never seen my printer run with such smoothness and precision.
I don't know anything about IS so it's at the default setting, and I ran the MPC PID and let it set the values.
Using my center-mount BLT Fang Duct system... https://www.printables.com/model/332248-fang-duct-with-center-mounting-bltouch-for-aquila-
Probing all 81 pints 
Thanks so much for all your hard work and timeless hours taken away from your life just so the world can print better. I can't even begin to give you the thanks you deserve.
https://www.youtube.com/watch?v=vY2IeydB2n4

**Comment by mriscoc** [comment]
Thanks, I saw the video, that was a nice print!
